util/bigint/src/hash.rs

```diff
@@ -62,6 +62,8 @@ pub fn clean_0x(s: &str) -> &str {
 	}
 }
 
+extern { fn memcmp(s1: *const i8, s2: *const i8, n: usize) -> i32; }
```

Comment:
`memcmp` is intrinsic and does not actually require libc. I'd prefer not to add a dependency that is not really needed.